WeakAuras/RegionTypes/RegionPrototype.lua

```diff
@@ -203,7 +203,8 @@ local function SoundPlay(self, options)
     return
   end
 
-  self:SoundStop();
+  local fadeoutTime = options.sound_fade and options.sound_fade * 1000 or 0
+  self:SoundStop(fadeoutTime);
```

Comment:
This should imho not use the fadeoutTime at all, but immediately stop.

Reply:
In what situation do you want the sound to stop immediately even if the user has set a sound fade in conditions?
Or are you saying there shouldn't be a sound fadeout in conditions?

Reply:
That isn't the code path from a SoundStop via conditions, this is when the user uses "Sound/Play" and we automatically stop the sound, in that case the sound should stop immediately and not be affected by an unrelated setting.

Reply:
Or actually you might be right that it is the code path called from options. Still the code here is wrong to assume that the settings apply. So there's a check for options.sound_type == "Stop" missing here.

Reply:
I added the check for sound_type == "Stop" in 76896a6 and changed the code a bit so that `soundFadeout` is set to `self` / `region` instead of a local variable.